README.md

```diff
@@ -5,12 +5,13 @@ The `:Luadev` command will open an scratch window which will show output from ex
 
 Use the folllowing mappings to execute lua code:
 
-Binding                   | Action
-------------------------- | ------
-`<Plug>(Luadev-RunLine)`  | Execute the current line
-`<Plug>(Luadev-Run)`      | in visual mode: execute visual selection
-`<Plug>(Luadev-RunWord)`  | Eval identifier under cursor, including `table.attr`
-`<Plug>(Luadev-Complete)` | in insert mode: complete (nested) global table fields
+Binding                       | Action
+-------------------------     | ------
+`<Plug>(Luadev-RunLine)`      | Execute the current line
+`<Plug>(Luadev-Run)`          | in visual mode: execute visual selection
```

Comment:
update the docs (just describe it as an operatior, the visual and normal-linewise behavior is _standard_ operator behavior after all)